src/float.rs

```diff
@@ -1493,8 +1493,8 @@ mod tests {
             assert!((FloatCore::to_radians(deg) - rad).abs() < 1e-6);
 
             let (deg, rad) = (deg as f32, rad as f32);
-            assert!((FloatCore::to_degrees(rad) - deg).abs() < 1e-6);
-            assert!((FloatCore::to_radians(deg) - rad).abs() < 1e-6);
+            assert!((FloatCore::to_degrees(rad) - deg).abs() < 1e-5);
+            assert!((FloatCore::to_radians(deg) - rad).abs() < 1e-5);
```

Comment:
Why was this necessary?

Reply:
After I made it forward to std's `f32::to_degrees`, it started failing on nightly. I suspect rust-lang/rust#47919, which tried to be more accurate, actually made things *less* accurate for these inputs, but I haven't investigated yet. I will probably file an upstream bug.

Reply:
https://github.com/rust-lang/rust/issues/48617